src/lib/modules/scaffolding/framework/react/index.ts

```diff
@@ -0,0 +1,101 @@
+import Handlebars from "handlebars";
+import * as path from "path";
+
+import { Contract } from "../../../../../typings/contract";
+import { Embark } from "../../../../../typings/embark";
+import { Builder } from "../../builder";
+import { Description } from "../../description";
+import { Options } from "../../options";
+
+const fs = require("../../../../core/fs");
+const utils = require("../../../../utils/utils");
+require("../../handlebarHelpers");
+
+const indexTemplatePath = path.join(__dirname, "templates", "index.html.hbs");
+const dappTemplatePath = path.join(__dirname, "templates", "dapp.js.hbs");
+
+export class React implements Builder {
```

Comment:
This another case where I'd highly advice against calling the class `React`, considering that not only such a thing exists already, but it's also very ambiguous. Similar to above, I'd vote for something like `ReactBuilder` or `ReactScaffolder` instead.

Reply:
renamed to ReactBuilder